zip-0317.rst

```diff
@@ -241,6 +264,49 @@ described above) to the original conventional fee of 10000 zatoshis
 specified in [#zip-0313]_, and imposes a non-trivial cost for
 potential denial-of-service attacks.
 
+ZSA Fee calculation
+-------------------
+In addition to the parameters defined in the `Fee calculation`_ section, the ZSA protocol upgrade defines the following additional parameter:
+
+===================================== =============== ===============================================
+Parameter                                 Value         Units
+===================================== =============== ===============================================
+:math:`issuance\_fee`                 :math:`1000000` zatoshis per issuance action (as defined below)
+:math:`finalize\_fee`                 :math:`1000000` zatoshis per issuance action (as defined below)
+===================================== =============== ===============================================
+
+Wallets implementing this specification SHOULD use a conventional fee
+calculated in zatoshis per the following formula:
+
+.. math::
+
+   \begin{array}{rcl}
+     zsa\_logical\_actions  &=& logical\_actions \;- \\
+                       & & nActionsOrchard \;+ \\
```

Comment:
Why does the formula subtract the number of Orchard actions (nActionsOrchard)?

This means that the non-ZSA Orchard actions are not charging any fee, which is a denial of service risk.

Reply:
It might be clearer to keep`nActionsOrchard` in the total (don't subtract it), and add a note that it includes both wrapped and native assets.

Reply:
Updated